cmd/backup-manager/app/clean/manager.go

```diff
@@ -102,3 +120,51 @@ func (bm *Manager) performCleanBackup(ctx context.Context, backup *v1alpha1.Back
 		Status: corev1.ConditionTrue,
 	}, nil)
 }
+
+// getNextBackup to get next backup sorted by start time
+func (bm *Manager) getNextBackup(ctx context.Context, backup *v1alpha1.Backup) (*v1alpha1.Backup, error) {
+	var err error
+	bks, err := bm.backupLister.Backups(backup.Namespace).List(labels.Everything())
+	if err != nil {
+		return nil, fmt.Errorf("get namespace %s backups list failure", backup.Namespace)
+	}
+
+	sort.Slice(bks, func(i, j int) bool {
+		return bks[i].Status.TimeStarted.Before(&bks[j].Status.TimeStarted)
+	})
+
+	for i, bk := range bks {
+		if backup.Name == bk.Name {
+			// only one backup or it is the last one in slice
+			if i >= len(bks)-1 {
+				return nil, nil
+			}
+			return bks[i+1], nil
+		}
+	}
+
+	return nil, fmt.Errorf("get next backup within namespace %s failure", backup.Namespace)
+}
+
+// updateBackupSize update a volume-snapshot backup size
+func (bm *Manager) updateBackupSize(ctx context.Context, backup *v1alpha1.Backup) error {
```

Comment:
Rename it to `updateVolumeSnapshotBackupSize`

Reply:
fixed in https://github.com/pingcap/tidb-operator/pull/4819/commits/fc18e21a421dee08cf3257099b503047e5d552a7